test/basic.js

```diff
@@ -54,7 +54,7 @@ tape('flush', function (t) {
 })
 
 tape('verify', function (t) {
-  t.plan(9)
+  t.plan(4)
```

Comment:
hey @denim2x this change looks like a leftover error from previous work ^_^  I think this should say 9 here?

I noticed this because it's me who wrote the old code that got changed.